release-please-config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "bootstrap-sha": "9903e0c129683979234f287d59f7468ca5cd7341",
+  "exclude-paths": [
+    ".git",
+    ".idea",
+    ".github",
+    ".vscode"
+  ],
+  "release-type": "dotnet-yoshi",
+  "include-component-in-tag": false,
+  "include-v-in-tag": true,
+  "packages": {
+    ".": {
+      "package-name": "src/Microsoft.Graph.Core/",
+      "changelog-path": "CHANGELOG.md",
+      "extra-files": [
+        "src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj",
```

Comment:
we probably need to add the comments in those files to RP knows where to make the update?